kolibri/content/utils/import_export_content.py

```diff
@@ -61,3 +77,40 @@ def get_num_coach_contents(contentnode, filter_available=True):
             .count()
     else:
         return 1 if contentnode.coach_content else 0
+
+
+def retry_import(command, e, name, f=None, overall_progress_update=None):
+    logging.error("An error occured during channel import: {}".format(e))
```

Comment:
Logging the exception in a different module than it occurred, makes it harder to troubleshoot.

You can think of the function `retry_import` as doing what it is named to do.. the logging part is beyond that, so it's more natural that the [callee](https://en.wikipedia.org/wiki/Calling_convention).

Reply:
ah yes that makes sense. I will move the log to the module that the exception occurs.